pkg/repo/callrepo.go

```diff
@@ -1,53 +1,30 @@
 package repo
 
 import (
+	"context"
 	"fmt"
 	"log/slog"
 	"sync"
 
+	"github.com/ksysoev/deriv-api-bff/pkg/config"
 	"github.com/ksysoev/deriv-api-bff/pkg/core"
 	"github.com/ksysoev/deriv-api-bff/pkg/core/composer"
 	"github.com/ksysoev/deriv-api-bff/pkg/core/handler"
 	"github.com/ksysoev/deriv-api-bff/pkg/core/processor"
 	"github.com/ksysoev/deriv-api-bff/pkg/core/validator"
 )
 
-type CallsConfig struct {
-	Calls []CallConfig `mapstructure:"calls"`
-}
-
-type EtcdConfig struct {
-	Servers            []string `mapstructure:"servers"`
-	DialTimeoutSeconds int      `mapstructure:"dialTimeoutSeconds"`
-}
-
-type CallConfig struct {
-	Method  string           `mapstructure:"method"`
-	Params  validator.Config `mapstructure:"params"`
-	Backend []*BackendConfig `mapstructure:"backend"`
-}
-
-type BackendConfig struct {
-	Name            string            `mapstructure:"name"`
```

Comment:
@shan-96 better to avoid centralizing configuration, we try to avoid cases where all packages depends on single config struct

Reply:
yep this is just a refactor into a single module to avoid cyclic dependency